DigitalLearningSolutions.Data/Models/TrackingSystem/EvaluationResponses.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DigitalLearningSolutions.Data.Models.TrackingSystem
+{
+    public class EvaluationResponses
+    {
+        public EvaluationResponses(string response, int count, int totalResponses)
+        {
+            Response = response;
+            Count = count;
+            Percentage = (float)count / totalResponses;

```

Comment:
Just realised this technically isn't a percentage. Could do with either being renamed or multiplying by 100 to make it a percentage.

Reply:
I'm not really sure what I should name this, it is technically a percentage, just written in decimal form. Googling for a more correct name comes up with that there is no correct name for this. e.g this [thread on Maths StackExchange](https://math.stackexchange.com/questions/1350980/correct-term-for-percentage-in-decimal-form).
Options really are Proportion (not really correct), Percentage (technically correct), DecimalPercentage (not a real term but does describe what we are using, or something more wordy to fully describe it. I've gone with DecimalPercentage for now, but am open to other thoughts.

Multiplying by 100 is incorrect here, it changes the meaning to 100x bigger.

Reply:
I think DecimalPercentage is suitable enough if the value stays as it is. 

This might be a stupid question, but why would multiplying by 100 be incorrect here? It would be converted to an actual percentage, and I don't think its used in any calculations. Is it necessary to keep it as a decimal?

Reply:
Ah right, I think I might have been getting confused with some view models where we do multiply by 100 and do some rounding for display. Fair point. In which case either Percentage or DecimalPercentage works as a name, but I'll approve this and leave it up to you if you have any preference.